docs/madr/decisions/080-inbound-policies-matches.md

```diff
@@ -0,0 +1,565 @@
+
+Though, without a label-enforcement mechanism, the feature isn’t truly secure: 
+if clients can add or remove any labels on their own workloads, they can simply insert themselves into the allow list.
+
+##### Verify user stories
+
+###### Mesh Operator
+
+1. I want all requests in the mesh to be denied by default (2.12)
+
+No MeshTrafficPermission policies means requests are denied by default.
+
+2. I want to declare a group of identities as explicitly denied,
+   so that Service Owner can't override or bypass that decision,
+   ensuring enforcement of critical security boundaries across the mesh. (2.12)
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-mesh-operator
+spec:
+   default:
+      deny:
+         - spiffeId:
+              type: Exact
+              value: "spiffe://trust-domain.mesh/ns/default/sa/api-gateway"
+         - spiffeId:
+              type: Exact
+              value: "spiffe://trust-domain.mesh/ns/default/sa/legacy-workload"
+         - spiffeId:
+              type: Prefix
+              value: "spiffe://legacy.mesh/"
+```
+
+3. I want to allow all clients in the `observability` namespace to access all services by default,
+   so that telemetry and monitoring tools function automatically,
+   while still allowing Service Owners to explicitly opt out by applying deny policies. (2.12)
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-mesh-operator
+spec:
+   default:
+      allow:
+         - spiffeId:
+              type: Prefix
+              value: "spiffe://trust-domain.mesh/ns/observability"
+```
+
+4. I want to allow all clients in the `observability` namespace to access the `/metrics` endpoint on all services,
+   so that monitoring tools can collect metrics without requiring each Service Owner to configure access individually.
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-mesh-operator
+spec:
+   default:
+      allow:
+         - spiffeId:
+              type: Prefix
+              value: "spiffe://trust-domain.mesh/ns/observability"
+           path:
+             type: Prefix
+             value: "/metrics"
+```
+
+###### Service Owner
+
+1. I want to grant access to my service to any client I choose,
+   so that I can support integrations and collaboration with other teams,
+   unless the Mesh Operator has explicitly denied that client,
+   ensuring I remain in control while respecting mesh-wide security boundaries. (2.12)
+
+```yaml
+type: MeshTrafficPermission
+mesh: default
+name: by-backend-owner
+spec:
+   targetRef:
+      kind: Dataplane
+      labels:
+         app: backend
+   default:
+      allow:
+         - spiffeId:
+              type: Prefix
+              value: "spiffe://trust-domain.mesh/"
+```
+
+2. I want to opt out of mesh-wide `observability` access,
+   by denying requests from the `observability` namespace,
+   so that my service’s sensitive endpoints remain private unless explicitly allowed. (2.12)
```

Comment:
Does this mean `spiffe://trust-domain.mesh/ns/observability` can still access `/metrics` or not? 🤔